src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-width;
+  margin: 10% auto;
+  padding: $calendar-pd;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 30px;
+  @for $index from 28 through $calendar-numberOf-day {
+    &--month-length--#{$index} >
+    &__day:nth-last-child(-n + #{31 - $index}) {
+      display: none;
+    }
+  }
+  @each $name, $value in $day-of-week {
+    &--start-day--#{$name} > &__day:first-child {
+      margin-left: #{$value};
+    }
+  }
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $day-height;
+    width: $day-width;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
```

Comment:
You should use in hover only the parameters which should be changed. Not cursor: pointer